packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3455,6 +3455,28 @@ class RenderSemanticsGestureHandler extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// If non-null, the look up table of the actions to their localized labels.
+  ///
+  /// Set by the [RawGestureDetectorState] to provide appropriate labels for
+  /// smeantics actions.
+  ///
+  /// This is used to generate action label overrides for scrolling semantics
```

Comment:
This is more than just override the action, it will add custom action to the child subtree but not itself. I fear to make this a general feature because this is more of a workaround to make iOS switch control feel native.